Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 git_source(:github) do |repo_name|
-  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  repo_name = '#{repo_name}/#{repo_name}' unless repo_name.include?('/')
```

Comment:
All variables interpolation must be in double quotation marks.
 
Eg.

`"#{repo_name}/#{repo_name}"`